Jans first commit

JanWien opened the pull request:
Classical Weiherbach files for landuse

Commit:
Jans first commit

Classical Weiherbach files for landuse

mmaelicke committed:
use whitespaces

mmaelicke committed:
Uniform parameter file names

mmaelicke commented:
@JanWien  I changed the files to use uniform file extensions and whitespaces instead of tabs. The validation report in fact is updated in place. Nice!

AlexDo1 commented on in/landuse/landuseclass.def:
```diff
@@ -0,0 +1,9 @@
+2        Coniferous Forest         in/landuse/Coniferous_forest.PAR
+3        Corn                      in/landuse/Corn.PAR  
+4        Crops & Vegetables        in/landuse/Crops_vegetables.PAR
+5        Deciduous_forest          in/landuse/Deciduous_forest.PAR
```
Classname `Deciduous_forest` should be `Deciduous Forest` to be consistent with Line 2.

AlexDo1 commented on in/landuse/landuseclass.def:
```diff
@@ -0,0 +1,9 @@
+5        Deciduous_forest          in/landuse/Deciduous_forest.PAR
+6        Indian Mustard            in/landuse/Indian_mustard.PAR
+7        Pasture                   in/landuse/Pasture.PAR
+8        Rapeseed + Field Mustard  in/landuse/Rapeseed_FieldMustard.PAR
```
In Line 4 you used `&` for the combination of two crops, here you used a `+` sign, again, for consistency we should decide for one (I vote for `&`)

AlexDo1 reviewed (changes requested):
Besides the two minor things I found in `landuseclass.def`, I only found some (encoding) and language errors in the comments in the files, but I guess we will get rid of the comments anyways, so you don't have to change them.

Commit:
Update landuseclass.def

Resolve Alex review comments: Class names

AlexDo1 commented:
Looking good now, maybe @Ash-Manoj can give his review and then we can merge.

JanWien commented:
I'll edit the comments first maybe?


Am 19.01.2023 um 16:29 schrieb Alexander Dolich:
>
> Looking good now, maybe @Ash-Manoj <https://github.com/Ash-Manoj> can 
> give his review and then we can merge.
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/CATFLOW/example-parameters/pull/6#issuecomment-1397157979>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AMTWGMSSQ7D7HSEFA74LX6LWTFMXHANCNFSM6AAAAAAUAERYOY>.
> You are receiving this because you were mentioned.Message ID: 
> ***@***.***>
>

-- 
  ________________________________________________________________

  Karlsruher Institut für Technologie (KIT)
  Institut für Wasser und Gewässerentwicklung
  Bereich Hydrologie
		
  Dr. Jan Wienhöfer
  Wissenschaftlicher Mitarbeiter
		
  Kaiserstraße 12, 76131 Karlsruhe
  Geb. 10.81, Raum: 423
		
  Tel.: +49 721 608 - 41932
  Web:http://hyd.iwg.kit.edu	
  ________________________________________________________________
  KIT – Die Forschungsuniversität in der Helmholtz-Gemeinschaft

AlexDo1 commented:
Sure, I was not sure if you want to edit the comments.

Commit:
Comments and class names

Edited comments to only contain the mentioned people (even if I don't know if this could be related to sources)
Renamed pasture to grassland, which I think is better

Ash-Manoj commented:
I had a quick look and it looks fine to me!

Ash-Manoj reviewed (commented):
All good!
Can proceed!
